lib/matplotlib/bezier.py

```diff
@@ -291,11 +291,9 @@ def split_path_inout(path, inside, tolerence=0.01, reorder_inout=False):
 
 
 def inside_circle(cx, cy, r):
-    r2 = r ** 2
-
     def _f(xy):
         x, y = xy
-        return (x - cx) ** 2 + (y - cy) ** 2 < r2
```

Comment:
I'm not sure that this change is an improvement. The original is probably faster, since there is a square-root computation inside hypot.

Reply:
the real cost is actually setting up the numpy ufunc, but sure.